cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-8183` - The CDAP UI now automatically retries loading the homepage
+  when the CDAP Server is not up and ready yet.
+
+- :cask-issue:`CDAP-8250` - Reduced non-informative stacktrace information in the log when
+  a connection to the CDAP Router is closed prematurely.
+
+- :cask-issue:`CDAP-8398` - The CDAP UI added a "Just Added" section to the home page.
+
+- :cask-issue:`CDAP-8565` - Improved the master process stop procedure to support fast
+  failover when running with HA. Added a new kill command to force-kill CDAP processes.
+
+- :cask-issue:`HYDRATOR-208` - The CDAP UI added a duration timer to CDAP pipelines.
+
+- :cask-issue:`HYDRATOR-282` - Updated the CSVParser plugin to change "PDL" to "Pipe
+  Delimited" and "TDF" to "Tab Delimited".
+
+- :cask-issue:`HYDRATOR-577` - Changed the Table sink plugin to make using the row key
+  easier.
+
+- :cask-issue:`HYDRATOR-1006` - Updated the Tokenizer plugin to be more forgiving when
+  parsing tokens.
+
+- :cask-issue:`HYDRATOR-1028` - Changed the Data Generator configuration to be easier to
+  use.
+
+- :cask-issue:`HYDRATOR-1073` - Added a "Fail This Pipeline" sink plugin in a repo at
+  `github.com/hydrator/failpipeline-sink <https://github.com/hydrator/failpipeline-sink>`__.
+
+- :cask-issue:`HYDRATOR-1074` - Added a plugin for fetching data from an external HTTP
+  site and writing the response to HDFS, available at
+  `github.com/hydrator/httptohdfs-action <https://github.com/hydrator/httptohdfs-action>`__.
+
+- :cask-issue:`HYDRATOR-1144` - Updated the use of "true/false" select boxes to be
+  consistent in their ordering.
+
+- :cask-issue:`HYDRATOR-1149` - Added the ability to read recursive directories to the
+  File source plugin.
+
+- :cask-issue:`HYDRATOR-1162` - Added logging to an error-dataset to the LogParser and
+  XMLMultiParser plugins.
+
+- :cask-issue:`HYDRATOR-1177` - Plugins can now retrieve the input and output schema of
+  their stage in their initialize methods.
+
+- :cask-issue:`HYDRATOR-1323` - Added a new ErrorTransform plugin-type that can be placed
+  after a pipeline stage to consume errors emitted by that stage.
+
+- :cask-issue:`WRANGLER-3` - The CDAP UI's Wrangler modal dialog will give a warning when
+  you try to close or exit out of it without confirmation.
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-2543` - Fixed an issue of a hanging application in the case that a
+  user program JAR is missing dependencies.
+
+- :cask-issue:`CDAP-4739` - Fixed an issue to make artifact, datasets, logs, and
+  coprocessor jar locations resilient to an HA upgrade.
+
+- :cask-issue:`CDAP-5107` - Added an HTTP RESTful endpoint to retrieve a specific property
+  for a specific version of an artifact in the 'system' scope.
+
+- :cask-issue:`CDAP-5717` - Fixed an issue with starting the CDAP CLI and the CDAP
+  Standalone when the on-disk path has a space in it.
+
+- :cask-issue:`CDAP-6642` - Attempting to delete a system artifact by specifying a user
```

Comment:
I think we should create a new section for these. If a response code changed (even if it is the right thing to do), it shouldn't be buried under bug fixes - perhaps "Incompatible Changes" or something to that effect. What do you think?